src/Core/Services/Implementations/HandlebarsMailService.cs

```diff
@@ -295,6 +294,20 @@ public async Task SendOrganizationUserRemovedForPolicyTwoStepEmailAsync(string o
         await _mailDeliveryService.SendEmailAsync(message);
     }
 
+    public async Task SendOrganizationUserRevokedForTwoFactoryPolicyEmailAsync(string organizationName, string email)
+    {
+        var message = CreateDefaultMessage($"You have been revoked from {organizationName}", email);
```

Comment:
Does the org name need to be sanitized here? (@r-tome you did all the sanitization stuff I think?)

Reply:
Thanks!